packages/backend-common/src/service/types.ts

```diff
@@ -98,6 +98,23 @@ export type ServiceBuilder = {
     requestLoggingHandler: RequestLoggingHandlerFactory,
   ): ServiceBuilder;
 
+  /**
+   * Sets an additional errorHandler to run before the defaultErrorHandler.
+   *
+   * If we want to use only custom errorHandler without defaultErrorHandler we need to
```

Comment:
Perhaps update the block in the middle with;
```
For execution of only the custom error handler make sure to also invoke disableDefaultErrorHandler() otherwise the defaultErrorHandler is executed at the end of the error middleware chain.
```

Reply:
I've made changes, had some problems with committing via Github UI, but it should look ok now